slices/base-files.yaml

```diff
@@ -36,7 +36,7 @@ slices:
       /var/cache/:
       /var/lib/:
       /var/log/:
-      /var/run/:
+      /var/run/: {symlink: /run}
```

Comment:
Given the mutation script's limitation and the fact that the corresponding maintainer script is quite deterministic about this, I think it is reasonable do have this symlink here.

Could you please add a comment for it though? Just so that users understand "why" this is a symlink instead of the regular folder as in the deb's contents.

Reply:
Yep, done.